rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -2232,6 +2232,41 @@ void getRetiringFederatorPublicKeyOfType(MessageCall.MsgType msgType, Activation
         }
     }
 
+    @ParameterizedTest()
+    @MethodSource("msgTypesAndActivations")
+    void getProposedFederationAddress(MessageCall.MsgType msgType, ActivationConfig activationConfig) throws VMException {
+        Transaction rskTxMock = mock(Transaction.class);
+        doReturn(true).when(rskTxMock).isLocalCallTransaction();
+
+        BridgeSupport bridgeSupportMock = mock(BridgeSupport.class);
+        Address expectedAddress = Address.fromBase58(networkParameters, "32Bhwee9FzQbuaG29RcXpdrvYnvZeMk11M");
+        when(bridgeSupportMock.getProposedFederationAddress()).thenReturn(Optional.of(expectedAddress));
+
+        Bridge bridge = bridgeBuilder
+            .transaction(rskTxMock)
+            .activationConfig(activationConfig)
+            .bridgeSupport(bridgeSupportMock)
+            .msgType(msgType)
+            .build();
+
+        CallTransaction.Function function = BridgeMethods.GET_PROPOSED_FEDERATION_ADDRESS.getFunction();
+        byte[] data = function.encode();
+
+        if (activationConfig.isActive(ConsensusRule.RSKIP419, 0)) {
+            if (activationConfig.isActive(ConsensusRule.RSKIP417, 0) &&
```

Comment:
Perhaps we could remove the check for RSKIP417 activation here? It's implied it will be active if RSKIP419 is active

Reply:
Thanks for spotting it, I have updated it.